### PR TITLE
feat(ServerListIndicator) add ids for css

### PR DESCRIPTION
**Pull request by Panniku**
Adds ids for the parent container and texts in this plugin, useful for targetting the container with css and hiding text when needed

### CODE REVIEW

**Comment by Panniku** [comment]
bump

File: src/plugins/serverListIndicators/index.tsx [hunk 1]
```diff
@@ -18,11 +18,14 @@
...
 import { Settings } from "@api/Settings";
+import { enableStyle } from "@api/Styles";
 import ErrorBoundary from "@components/ErrorBoundary";
 import { Devs } from "@utils/constants";
 import { useForceUpdater } from "@utils/react";
 import definePlugin, { OptionType } from "@utils/types";
-import { GuildStore, PresenceStore, RelationshipStore } from "@webpack/common";
+import { GuildStore, PresenceStore, RelationshipStore, Tooltip } from "@webpack/common";
+
+import style from "./styles.css?managed";
```
Comment (Vendicated) [inline src/plugins/serverListIndicators/index.tsx:28]: there's no need for this to be managed! you only need to manually manage your styles when they change styling of stock discord ui
```suggestion
import "./styles.css";
```

File: src/plugins/spotifyControls/PlayerComponent.tsx [hunk 2]
```diff
@@ -339,6 +340,25 @@ function Info({ track }: { track: Track; }) {
     );
 }
 
+async function getDominantColor(imageUrl: any) {
```
Comment (Vendicated) [inline src/plugins/spotifyControls/PlayerComponent.tsx:343]: please only include one change per pr. this change should not be here

**Review by Vendicated — COMMENTED** [review]
thanks for your contribution! could you please show a screenshot?

File: src/plugins/spotifyControls/PlayerComponent.tsx (hunk 2 above)
Comment (Panniku) [reply src/plugins/spotifyControls/PlayerComponent.tsx:343]: sorrry, accidentally merged changes when attempting to sync repo

**Comment by Panniku** [comment]
> thanks for your contribution! could you please show a screenshot?

Screenshot
image

File: src/plugins/spotifyControls/PlayerComponent.tsx [hunk 3]
```diff
@@ -375,10 +376,17 @@ export function Player() {
     } as React.CSSProperties;
 
     return (
-        <div id={cl("player")} style={exportTrackImageStyle}>
-            <Info track={track} />
-            <SeekBar />
-            <Controls />
-        </div>
+        <ErrorBoundary fallback={() => (
```
Comment (Cynosphere) [inline src/plugins/spotifyControls/PlayerComponent.tsx:379]: This added itself back from the revert

**Review by Cynosphere — CHANGES_REQUESTED** [review]

**Pull request closed without merging**